src/pages/fareConfirmation.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { ReactElement } from 'react';
+import upperFirst from 'lodash/upperFirst';
+import { FARE_TYPE_ATTRIBUTE, PASSENGER_TYPE_ATTRIBUTE, TIME_RESTRICTIONS_DEFINITION_ATTRIBUTE } from '../constants';
+import { CustomAppProps, NextPageContextWithSession, TimeRestriction } from '../interfaces';
+import TwoThirdsLayout from '../layout/Layout';
+import CsrfForm from '../components/CsrfForm';
+import ConfirmationTable, { ConfirmationElement } from '../components/ConfirmationTable';
+import { getSessionAttribute } from '../utils/sessions';
+import { isPassengerTypeAttributeWithErrors, isFareTypeAttributeWithErrors } from '../interfaces/typeGuards';
+import { PassengerType } from './api/passengerType';
+
+const title = 'Fare Confirmation - Fares Data Build Tool';
+const description = 'Fare Confirmation page of the Fares Data Build Tool';
+
+type FareConfirmationProps = {
+    fareType: string;
+    passengerType: PassengerType;
+    timeRestrictions: TimeRestriction;
+};
+
+export const buildFareConfirmationElements = (
+    fareType: string,
+    passengerType: PassengerType,
+    timeRestrictions: TimeRestriction,
+): ConfirmationElement[] => {
+    const confirmationElements: ConfirmationElement[] = [
+        {
+            name: 'Fare Type',
+            content: upperFirst(fareType),
+            href: 'fareType',
+        },
+        {
+            name: 'Passenger Type',
+            content: upperFirst(passengerType.passengerType),
+            href: 'passengerType',
+        },
+    ];
+    if (passengerType.ageRange && (passengerType.ageRangeMin || passengerType.ageRangeMax)) {
```

Comment:
would we able to destructure these variables

Reply:
Unsure what you mean here - do you mean have them as variables? If so, I'm not sure what it adds.